### PR TITLE
improve set_build.py, pin torba version

### PR DESCRIPTION
**Pull request by jackrobison**
- adds BUILD_COMMIT to lbrynet.build_type

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,10 +30,9 @@ jobs:
         - bash <(curl -s https://codecov.io/bash)
 
     - name: "Integration Tests"
-      cache: false
       install:
         - pip install tox-travis coverage
-        - pushd .. && git clone --single-branch --branch master https://github.com/lbryio/torba.git && popd
+        - pip install git+https://github.com/lbryio/torba.git@${TORBA}#egg=torba
```
Comment (eukreign) [inline .travis.yml]: this doesn't make sense, why did you change it from checking out the correct version to install with pip?

Comment (jackrobison) [reply .travis.yml]: it installs the same version by default - `master` - determined by the `TORBA` travis variable

Comment (eukreign) [reply .travis.yml]: install it into what?
